dictionaryutils/__init__.py

```diff
@@ -239,3 +239,19 @@ def resolve_schema(self, obj, root):
             return [self.resolve_schema(item, root) for item in obj]
         else:
             return obj
+
+    def allow_nulls(self):
+        for key, value in self.schema.items():
+            try:
+                try:
+                    required = value["required"]
+                except KeyError:
+                    required = []
+                for k, v in value["properties"].items():
+                    if k not in required and "type" in v:
```

Comment:
in which case would `type` not be specified?

Reply:
Some of the properties do not have a type field (at least in the dictionary that is in the sheepdog test). For example the properties `updated_datetime`, `treatment_anatomic_site`, and `treatment_outcome` for the treatment.yaml node.